ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -216,6 +246,34 @@ private MessageWriteSetStateInStore checkWriteSetStateInStore(MessageWriteSet me
     }
   }
 
+  /**
+   * Checks the used capacity of the store against the configured percentage thresholds to see if the store
+   * should be read-only or read-write
+   * @param totalCapacity total capacity of the store in bytes
+   * @param usedCapacity total used capacity of the store in bytes
+   */
+  private void checkCapacityAndUpdateHelix(long totalCapacity, long usedCapacity) {
```

Comment:
could we not refer to "Helix" in the blob store? Instead use "ClusterManager"?

Reply:
changed